# don't install /usr/share/icons/ubuntu-mono-{dark,light}
# it's only used in conjunction with appindicator
# and clutters the icon-themes menu in gnome
--- zim-0.60/setup.py-orig	2013-05-12 14:41:51.212603798 +0200
+++ zim-0.60/setup.py	2013-05-12 14:42:09.646172372 +0200
@@ -94,12 +94,6 @@
 			files = [os.path.join(dir, f) for f in files]
 			data_files.append((target, files))
 
-	# mono icons -> PREFIX/share/icons/ubuntu-mono-light | -dark
-	for theme in ('ubuntu-mono-light', 'ubuntu-mono-dark'):
-		file = os.path.join('icons', theme, 'zim-panel.svg')
-		target = os.path.join('share', 'icons', theme, 'apps', '22')
-		data_files.append((target, [file]))
-
 	# data -> PREFIX/share/zim
 	for dir, dirs, files in os.walk('data'):
 		if '.zim' in dirs:
